pkg/cloud-provider/cloudapi/aws/aws_converters.go

```diff
@@ -101,16 +109,24 @@ func convertFromSecurityGroupPair(cloudGroups []*ec2.UserIdGroupPair, managedSGs
 
 		cloudResourceIDs = append(cloudResourceIDs, cloudResourceID)
 	}
-	return cloudResourceIDs
+	return cloudResourceIDs, desc
 }
 
 // convertFromIPPermissionToIngressRule converts cloud ingress rules from ec2.IpPermission to internal securitygroup.IngressRule.
+// Each AT Sg can have one or more ANPs and an ANP can have one or more rules. Each rule can have a description.
 func convertFromIPPermissionToIngressRule(ipPermissions []*ec2.IpPermission, managedSGs map[string]*ec2.SecurityGroup,
 	unmanagedSGs map[string]*ec2.SecurityGroup) []securitygroup.IngressRule {
 	var ingressRules []securitygroup.IngressRule
 	for _, ipPermission := range ipPermissions {
-		fromSrcIPs := convertFromIPRange(ipPermission.IpRanges)
-		for _, srcIP := range fromSrcIPs {
+		fromSrcIPs, desc := convertFromIPRange(ipPermission.IpRanges)
+		for i, srcIP := range fromSrcIPs {
+			// Get cloud rule description.
+			_, err := securitygroup.ExtractCloudDescription(desc[i])
+			if err != nil {
+				// Ignore rules that don't have a valid description field.
+				awsPluginLogger().Error(err, "failed to extract cloud rule description", "desc", desc[i])
```

Comment:
log as debug, and will it be printed on every sync interval

Reply:
Ok will do.

Reply:
Changed the log level.